### ISSUE TITLE
its just one week and thats it

### ISSUE THREAD

**Issue by VannyTheBunny09** [luckydog7/Funkin-android#458]
dude whats taking so long. its just one week?

**Comment by VannyTheBunny09** [comment]
im so freaking pissed right now >:(

**Comment by WyxosYT** [comment]
> dude whats taking so long. its just one week? i'm so pissed right now

Dude can u shut up, the source code ain't even fucking out, have some patience man. And I'm also porting week 7 to android but I probably won't be able to publish it until week 7 fully comes out and then I can publish it to avoid my channel being taken down.
![Captura_de_Ecra_68.png](https://user-images.githubusercontent.com/82682892/115665409-30ad6500-a33b-11eb-98f8-33cecc152903.png)
![Captura_de_Ecra_67.png](https://user-images.githubusercontent.com/82682892/115665424-34d98280-a33b-11eb-8164-543cddac9393.png)
![Captura_de_Ecra_70.png](https://user-images.githubusercontent.com/82682892/115665445-3c009080-a33b-11eb-9a2e-c18c83078f9d.png)
![Screenshot_20210420_213308_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82682892/115665483-46bb2580-a33b-11eb-8918-ba871ebfe284.jpg)

**Comment by MrTks216** [comment]
Bruh, just one week? You do it for yourself. Do you think is easy to do? pls wait okay.

**Comment by WyxosYT** [comment]
> Bruh, just one week? You do it for yourself. Do you think is easy to do? pls wait okay.

Well yeah it is easy but when it comes to actually doing something it's gonna be hard and ur gonna meet alot of errors lmao

Like the "Expected {" I always get this Lmao

And also free broken practice mode code from me for no reason:
if (!practice_isenabled){
            function noteMiss(direction:Int = 1):Void
                {
                    if (!boyfriend.stunned)
                    {
                        health -= 0;
                        if (combo > 5 && gf.animOffsets.exists('sad'))
                        {
                            gf.playAnim('sad');
                        }
                        combo += 0;
            
                        songScore -= 0;
            
                        FlxG.sound.play(Paths.soundRandom('missnote', 1, 3), FlxG.random.float(0.1, 0.2));
                        // FlxG.sound.play(Paths.sound('missnote1'), 1, false);
                        // FlxG.log.add('played imss note');
            
                        boyfriend.stunned = false;
            
                        // get stunned for 5 seconds
                        new FlxTimer().start(0 / 0, function(tmr:FlxTimer)
                        {
                            boyfriend.stunned = false;
                        });

**Comment by MrTks216** [comment]
Well, good luck to fix it, idk about coding.

**Comment by WyxosYT** [comment]
> Well, good luck to fix it, idk about coding.

Well yes I did that code at 6am lmfoa and I also fixed the background from tiny ass shet to normal at 5am
![Screenshot_20210421_220440_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82682892/115667437-bfbb7c80-a33d-11eb-961b-1145dbb876a9.jpg)
![Captura_de_Ecra_67.png](https://user-images.githubusercontent.com/82682892/115667449-c34f0380-a33d-11eb-959f-410071f3d4dc.png)

**Comment by VRtoyboi** [comment]
Still waiting for the source code

**Comment by FridayNightSans** [comment]
> dude whats taking so long. its just one week? i'm so pissed right now

But did you think for lucky?

**Comment by Crunchyyy456** [comment]
> dude whats taking so long. its just one week? i'm so pissed right now

If its one week THEN GO FUCKING CODE IT YOU TWAT, be patient fuckface

**Comment by WyxosYT** [comment]
Also guys I'm kinda close to finishing the custom stage lmao![Screenshot_20210422_214417_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82682892/115782959-2e3b2180-a3b4-11eb-99f6-259e66980c42.jpg)

**Comment by WyxosYT** [comment]
https://youtu.be/X5J-5K0GxY8
First preview of my port out lol

**Comment by MrTks216** [comment]
> https://youtu.be/X5J-5K0GxY8
> First preview of my port out lol

oh nice, but still not perfect, at least good job coding week 7.👏

**Comment by WyxosYT** [comment]
> > https://youtu.be/X5J-5K0GxY8
> > First preview of my port out lol
> 
> oh nice, but still not perfect, at least good job coding week 7.👏

Yuh I messed up the animations on tankman lmao, it's supposed to be left but it's right instead xddf
Also ty it took me days;-;
Since I only have 3 hours of average per day to use my laptop._.

**Comment by MrTks216** [comment]
> > > https://youtu.be/X5J-5K0GxY8
> > > First preview of my port out lol
> > 
> > oh nice, but still not perfect, at least good job coding week 7.👏
> 
> Yuh I messed up the animations on tankman lmao, it's supposed to be left but it's right instead xddf
> Also ty it took me days;-;
> Since I only have 3 hours of average per day to use my laptop._.

its okay, get some rest, i hope you can do it better next time.👍

**Comment by WyxosYT** [comment]
> > > > https://youtu.be/X5J-5K0GxY8
> > > > First preview of my port out lol
> > > 
> > > oh nice, but still not perfect, at least good job coding week 7.👏
> > 
> > Yuh I messed up the animations on tankman lmao, it's supposed to be left but it's right instead xddf
> > Also ty it took me days;-;
> > Since I only have 3 hours of average per day to use my laptop._.
> 
> its okay, get some rest, i hope you can do it better next time.👍

"Next time?" What do u mean lol I haven't finished it yet, I need to fix alot of stuff add tankrolling the spectators and try to make the smoke work and the watchtower animate, plus fix the incorrect positions of the bg
And add practice mode since I removed it cuz it was making me get the _Expected { _error code lmao

**Comment by MrTks216** [comment]
> > > > > https://youtu.be/X5J-5K0GxY8
> > > > > First preview of my port out lol
> > > > 
> > > > oh nice, but still not perfect, at least good job coding week 7.👏
> > > 
> > > Yuh I messed up the animations on tankman lmao, it's supposed to be left but it's right instead xddf
> > > Also ty it took me days;-;
> > > Since I only have 3 hours of average per day to use my laptop._.
> > 
> > its okay, get some rest, i hope you can do it better next time.👍
> 
> "Next time?" What do u mean lol I haven't finished it yet, I need to fix alot of stuff add tankrolling the spectators and try to make the smoke work and the watchtower animate, plus fix the incorrect positions of the bg
> And add practice mode since I removed it cuz it was making me get the _Expected {_ error code lmao

Yeah, i know. I mean hope you next time you make more perfect than this time. Sry my bad lol

**Comment by WyxosYT** [comment]
Here's a week 7 preview;
![Screenshot_20210424_163920_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82682892/115969516-2f478c80-a535-11eb-98fc-6146475a758b.jpg)
https://youtu.be/Kpr2uJB231Y
:D

**Comment by MrTks216** [comment]
> Here's a week 7 preview;
> ![Screenshot_20210424_163920_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82682892/115969516-2f478c80-a535-11eb-98fc-6146475a758b.jpg)
> https://youtu.be/Kpr2uJB231Y
> :D

Damn, rn its so good, more details, more stuff, great job👍
